packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -332,18 +333,24 @@ export const actionCreators: ActionCreators = {
   },
     pluginRegistryUrl: string | undefined,
   ): AppThunk<KnownAction, Promise<IDevWorkspace>> => async (dispatch, getState): Promise<IDevWorkspace> => {
+
+    const state = getState();
+
+    if (state.dwPlugins.defaultEditorError) {
+      const message = `Not able to create workspace due required resources failed to load: ${state.dwPlugins.defaultEditorError}`;
```

Comment:
Same comment as above,

I wonder if it makes sense to change this to be something like:
```suggestion
      const message = `Required sources failed when trying to create the workspace: ${state.dwPlugins.defaultEditorError}`;
```
or
```suggestion
      const message = `Unable to create the workspace. Requesting default editor failed: ${state.dwPlugins.defaultEditorError}`;
```